projects/plugins/jetpack/_inc/lib/class-jetpack-tweetstorm-helper.php

```
@@ -392,7 +392,12 @@ private static function add_text_to_tweets( $block ) {
 		} else {
 			$lines = array( $block['text'] );
 		}
-		$line_total = count( $lines );
+
+		if ( is_countable( $lines ) ) {
```

Given lines 391 and 393, it seems unlikely it will ever not be a countable in a version of PHP where `count()` cares about being passed a non-countable. PHP 8 will throw or fatal from the `explode()` rather than returning null or false, while in earlier versions it might but `count()` at most only logs a deprecation warning.

But OTOH this doesn't really hurt anything, especially if reduced to be just one line instead of five.